piccolo/apps/schema/commands/generate.py

```diff
@@ -63,7 +63,10 @@ class RowMeta:
     table_name: str
     character_maximum_length: t.Optional[int]
     data_type: str
-
+    numeric_precision: t.Optional[t.Union[int, str]]
+    numeric_scale: t.Optional[t.Union[int, str]]
```

Comment:
Just out of interest, under what situations do we get a string back instead of an int?

I'm just trying to understand what edge cases there are.

Reply:
@dantownsend Good question. Oddly, I cannot trace this now in the docs, but I was operating under the impression that the values for ``precision`` and ``scale`` could sometimes be stored internally as a binary string -- which accounts for that extra cast to ``int``. Happy to remove . . .

Reply:
No, I'm happy to keep it - was just curious.